.github/workflows/create-cli-release.yml

```
@@ -19,7 +19,10 @@ on:
 jobs:
   check-for-moratorium:
     if: fromJSON(inputs.isStableCandidate)
-    uses: ./.github/workflows/ctc.yml
+    run: ./scripts/release/tps_lock_check cli ${{ github.sha }}
```

```suggestion
    run: ./scripts/release/tps_check_lock cli ${{ github.sha }}
```

i think there's a typo here in referencing the `/release/tps_check_lock` script

---

Great catch 🏆